templates/skeleton-of-thought/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "skeleton-of-thought"
+version = "0.0.1"
+description = ""
```

Comment:
quick reminder to add the new fields. Will add a CI check to reject templates that are missing a description and/or the `tool.templates-hub` sections